src/components/AdequacyDoughnut/AdequacyDoughnut.tsx

```diff
@@ -61,8 +61,8 @@ export let AdequacyDoughnut = withStore('adequacies')<Props>(({ serviceAreas, st
         <th>Inadequate Access</th>
       </tr>
       <tr>
-        <td>{formatNumber(numAdequatePopulation)} ({percentAdequatePopulation}%)</td>
-        <td>{formatNumber(numInadequatePopulation)} ({percentInadequatePopulation}%)</td>
+        <td>{formatNumber(numAdequatePopulation)} ({formatPercentage(percentAdequatePopulation)}%)</td>
+        <td>{formatNumber(numInadequatePopulation)} ({formatPercentage(percentInadequatePopulation)}%)</td>
```

Comment:
Nice

Reply:
Should the unit (%) be part of what formatPercentge does?

Reply:
Done